style.css

```diff
@@ -186,7 +185,7 @@
 
 .btn {
```

Comment:
In order to have a great UI and the best UX, it is recommended to implement the button interactions (enable, hover, etc.).
![image](https://user-images.githubusercontent.com/83596135/165347734-4361a686-8a75-4e9b-8cb4-fb0eee49b02e.png)

- [ ] Kindly, implement the button interactions in order to meet the project requirements;